### DISCUSSION TITLE
RAM consumption limited to 2 GB?

### DISCUSSION THREAD

**Discussion by 0xDedSec42** [photoprism/photoprism#4568] [Q&A] [answered]
Yesterday I uploaded a batch of about 800 photos to my Photoprism instance. I'm running Photoprism on an instance at Oracle Cloud with their ARM CPU. Photoprism runs just fine although the docker-compose.yaml states that it's made for AMD64.

Photoprism is using all four cores but only about 2 GB RAM although the instance has 24 GB RAM and Photoprism is the only application on that instance. Is this normal behaviour? Would it speed up the post processing using more RAM?

**Comment by lastzero** [comment, marked as answer]
That's funny! Our users normally ask how to LIMIT memory usage claiming that more memory is being used than expected, e.g:

- https://github.com/photoprism/photoprism/issues/633
- https://github.com/photoprism/photoprism/discussions/4543

If you have some memory left that you want to use, you can increase the MariaDB cache size as described here in our docs:

- https://docs.photoprism.app/getting-started/troubleshooting/performance/#mariadb

**Reply by 0xDedSec42** [reply]
Yeah, RAM really isn't an issue in my case :-). I will try fiddling around a little bit with the MySQL settings.
